### ISSUE TITLE
webhippie/php-caddy:5 no longer available?

### ISSUE THREAD

**Issue by brooksambrose** [dockhippie/php#9]
```
$ docker pull webhippie/php-caddy:5
Error response from daemon: manifest for webhippie/php-caddy:5 not found
```

**Comment by tboerger** [comment]
There is an error within the build, I need to fix that. I will post an update on this issue when it's resolved.

**Comment by tboerger** [comment]
Solved via 921cb02

**Comment by tboerger** [comment]
Thank you for the report, totally forgot this.
